server/src/main/java/com/project/server/controllers/TrivyController.java

```diff
@@ -41,24 +41,18 @@
     
        @Autowired
        private MailProperties mailProperties;
-    @PostMapping("/repo")
-    public void privRepoauthentication( @RequestHeader("login") String login, @RequestHeader("psw") String psw,@RequestHeader("repo") String repo, @RequestHeader("header") String version) throws IOException, NoSuchFieldException, SecurityException, IllegalArgumentException, IllegalAccessException
-    {
+    @PostMapping("/trivy")
+    public void privRepoauthentication( @RequestBody Map<String, String> json) throws IOException, NoSuchFieldException, SecurityException, IllegalArgumentException, IllegalAccessException
```

Comment:
O wiele lepiej byłoby stworzyć klasę, która miałaby pola, które was tu interesują i jeśli niektóre są wymagane, a niektóre opcjonalne, to wystarczy użyć odpowiednich typów albo dekoratorów i będziecie mieli od razu walidację jsona, który przychodzi w zapytaniu.